e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -382,7 +384,7 @@ def _get_climo_dataset(self, season: str) -> xr.Dataset:
             using other datasets.
         """
         filepath = self._get_climo_filepath(season)
-        ds = xr.open_dataset(filepath, use_cftime=True)
```

Comment:
I think the tests are failing now in the CI/CD build because we always add "X" and "Y" bounds if they are missing.

The [test results](https://github.com/E3SM-Project/e3sm_diags/actions/runs/7590599128/job/20677384186) indicate bounds are missing for the expected objects. If you're up for it, this is a good opportunity for you to try to fix the 11 failing unit tests to get some experience with pytest.

You have a few options to fix the tests:
1. Update the input xr.Dataset objects to have `lat_bnds` and `lon_bnds`
   - [Example](https://github.com/E3SM-Project/e3sm_diags/blob/f171083984beebb6fd41214458ddae82fdba6caf/tests/e3sm_diags/driver/utils/test_dataset_xr.py#L218-L345)
2. Update the expected `xr.Dataset` objects to have `lat_bnds` and `lon_bnds` 
   - [Example](https://github.com/E3SM-Project/e3sm_diags/blob/f171083984beebb6fd41214458ddae82fdba6caf/tests/e3sm_diags/driver/utils/test_dataset_xr.py#L367)